sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/ControlFlowGraph/CSharpControlFlowGraphBuilder.cs

```diff
@@ -1256,10 +1256,15 @@ private Block BuildCondition(ExpressionSyntax expression, Block trueSuccessor, B
                             BuildCondition(binaryExpression.Right, trueSuccessor, falseSuccessor));
 
                     case SyntaxKind.LogicalAndExpression:
+                        return BuildCondition(

```

Comment:
please verify existing tests and most likely add some new tests for both LogicalAnd and NullCoalescing expressions in `ControlFlowGraphTest.cs`